ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -59,4 +60,63 @@ public struct ShadowInfo: Equatable {
 
     /// The vertical offset of the shadow for shadow 2.
     public let yTwo: CGFloat
+
+    /// The number that the figma blur needs to be divided by to properly display shadows
```

Comment:
Can you add a link to this documentation: https://github.com/microsoft/apple-ux-guide/blob/gh-pages/Shadows.md

Reply:
Also might want to update the comment now not to say "divide". Maybe just "adjusted by"

Reply:
Thanks for catching that!